### PR TITLE
Add DORA knowledge to taxonomy

### PR DESCRIPTION
**Pull request by lubuatibm**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- Knowledge for the model about DORA (Digital Operational Resilience Act)
- Part of a InstructLab Test

**Input given at the prompt**

```
What is DORA about?
```

**Response from the original model**

```
DORA, or the DevOps Research and Assessment, is a set of key performance metrics for DevOps teams that helps evaluate their efficiency and effectiveness in delivering high-quality software. The acronym DORA stands for the names of its four founders: Jennifer Petoff, Jez Humble, Gene Kim, and Allspaw.
```

**Response from the fine-tuned model**

```
DORA is an acronym for Digital Operational Resilience Act, which is a set of regulations targeting the digital resilience of financial entities such as banks and insurance companies.
```

**Contribution checklist**

- [x ] The contribution was tested with `ilab generate`
- [ x] No errors or warnings were produced by `ilab generate`
- [x ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/banking/DORA/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      DORA establishes technical requirements for financial entities and ICT providers across four domains.
+      These domains are ICT risk management and governance, incident response and reporting, digital
+      operational resilience testing, and third-party risk management.
+      While information sharing is encouraged, it is not mandatory.
+      The requirements will be enforced proportionately, meaning smaller entities will not be held to the
+      same standards as major financial institutions. Although the Regulatory Technical Standards (RTSs)
+      and Implementing Technical Standards (ITSs) for each domain are still under development,
+      the existing DORA legislation provides some insight into the general requirements.
+
+#TODO
```
Comment (jjasghar) [inline knowledge/banking/DORA/qna.yaml:93]: This is the incorrect place, we can't accept this, we need to documentation.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Due to legal reasons, we can't accept IBM documentation or things that aren't in the [approved list](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md) until the Red Hat Legal approves these.

File: knowledge/banking/DORA/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+document:
+  repo: https://github.com/juliadenham/oscars2024_knowledge.git
```
Comment (bjhargrave) [inline knowledge/banking/DORA/qna.yaml:95]: If you are making a knowledge contribution, you should then be providing your knowledge markdown files in a git repo. This is the link to some other knowledge contribution's git repo holding markdown files.

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**